soc/arm/nordic_nrf/nrf53/Kconfig.defconfig.nrf5340_CPUAPP_QKAA

```diff
@@ -11,4 +11,12 @@ config SOC
 config NUM_IRQS
 	default 69
 
+config NRF5340_CPUAPP_ERRATUM19
+	bool "Include workarounds for Erratum 19"
+	default y
+	help
+	  This MUST be enabled if there is a chance the code will be run on
```

Comment:
Should we mention that this errata workaround also will work on devices that has fixed this errata, with the cost of a bigger flash size?

Reply:
Done